controllers/metrics.go

```diff
@@ -66,9 +96,28 @@ var (
 		},
 		drpolicySyncIntervalMetricLabelNames,
 	)
+
+	lastSyncDuration = prometheus.NewGaugeVec(
+		prometheus.GaugeOpts{
+			Name:      LastSyncDurationSeconds,
+			Namespace: metricNamespace,
+			Help:      "Duration of max sync time in seconds",
+		},
+		syncDurationMetricLabelNames,
+	)
+
+	lastSyncDataBytes = prometheus.NewGaugeVec(
+		prometheus.GaugeOpts{
+			Name:      LastSyncDataBytes,
+			Namespace: metricNamespace,
+			Help:      "Total data synced in bytes since last sync",
+		},
+		syncDataBytesMetricLabels,
+	)
 )
 
-func SyncPolicyMetricLables(drPolicy *rmn.DRPolicy, drpc *rmn.DRPlacementControl) prometheus.Labels {
+// lastSyncTime metrics
```

Comment:
The doc is not clear and not standard. Can you describe what this does?

Reply:
updated the doc text